packages/hypercontroller/src/jwt.ts

```diff
@@ -14,15 +14,15 @@ const forceHS512Verify = (jwtOptions: any) =>
     : { algorithms: ['HS512'] }
 
 // decorator creator
-const createAuth = (verify: any, { loader, bearer }: any) => {
+const createAuth = (verify: any, { loader, bearer, authCookieName }: any) => {
   const permit = new Bearer(
     bearer || {
       query: 'access_token',
     }
   )
 
   return async (req: Request) => {
-    const token = permit.check(req)
+    const token = permit.check(req) || req.cookies[authCookieName]
```

Comment:
I believe if `authCookieName` is undefined/null it means that the user didn't want this feature -- so this logic should check for it